.github/workflows/postman-testing.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   run-tests:
-    runs-on: [self-hosted, ubuntu-22.04, X64, medium]
```

Comment:
These shouldn't work, they large runners are missing the `ubuntu-22.04` still so need `Linux` for now temporarily, same for the other large runners below.

Reply:
It's ubuntu-22.04 but the labels are based on what's [here](https://github.com/Consensys/linea-monorepo/actions/runners?tab=self-hosted).

Reply:
I reverted this change